documentation/src/docs/deployment/run-local.md

```diff
@@ -71,68 +72,59 @@ wget https://raw.githubusercontent.com/pravega/pravega/master/docker/compose/doc
 
-1. Ensure that the server's certificate is trusted. If you run `/gradlew startStandalone` without it, you'll encounter the following error:
+    **Direct Trust:** This type of trust is established by adding the server's certificate to the truststore. It is a non-standard way of establishing trust when using self-signed certificates. If you using self-signed certificates such as those provided by Pravega for standalone mode, ensure that the server's certificate is in the JVM's truststore.
 
-```java
-Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
-        at sun.security.validator.PKIXValidator.doBuild(PKIXValidator.java:397)
-        at sun.security.validator.PKIXValidator.engineValidate(PKIXValidator.java:302)
-        at sun.security.validator.Validator.validate(Validator.java:260)
-        at sun.security.ssl.X509TrustManagerImpl.validate(X509TrustManager
-```
-2. To ensure the server's certificate is trusted, import it into the JVM's truststore. The following command sequence is used (in Linux) with the provided certificate file "cert.pem":
+The following command sequence is used (in Linux) with the provided certificate file `cert.pem` into the JVM's system truststore.
+
+   - `cd /path/to/pravega/config`
+   - Convert the `cert.pem` file to `DER` format: `openssl x509 -in cert.pem -inform pem -out cert.der  -outform der`
+   - Import the certificate into the local JVM's trust store:
+    `sudo keytool -importcert -alias local-CA -keystore /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts  -file cert.der` (using the default password `changeit`)
 
- - `cd /path/to/pravega/config`
- - Convert the 'cert.pem' file to `DER` format: `openssl x509 -in cert.pem -inform pem -out cert.der -outform der`
- - Import the certificate into the local JVM's trust store:
-  `sudo keytool -importcert -alias local-CA -keystore /usr/lib/jvm/java-8-openjdk-amd64/jre/lib/security/cacerts  -file cert.der` (using the default password 'changeit')
+   **Note:** If you want to use a custom truststore instead of adding the certificate to the system truststore, create a new truststore using Java keytool utility, add the certificate to it and configure the JVM to use it by setting the system properties `javax.net.ssl.trustStore` and `javax.net.ssl.trustStorePassword`.
 
 3. Run Pravega standalone mode using command `./gradlew startStandalone`.
 
 4. Verify that controller REST API is returning response over SSL/TLS:
 
     ```java
-     curl -v -k https://104.215.152.115:9091/v1/scopes
+      curl -v -k https://104.215.152.115:9091/v1/scopes
```

Comment:
Please generalize this URL to something like (`https://<host-ip>:....)) to avoid putting the specific IP address.